### ISSUE TITLE
[Request]  Build much smarter Agent profiles

### ISSUE THREAD

**Issue by BaronRK** [lobehub/lobe-chat#2743]
### 🥰 Feature Description

As it stands, creating new 'Agents' is confusing, clunky, and time-consuming.

### 🧐 Proposed Solution

This should be super simple:

- Click [Add Agent]
- Perhaps offer a feature to use one or more existing agents as a template to build on.
- Instantly set pic, colours, name.

Allow this to be edited just as easily, without forcing people back to Settings.

[I really have to ask, why is it so confusing, primitive, and wonky right now?]

  

### 📝 Additional Information

This is a standard CRUD issue, and for some reason, that is not being used, and instead Agent Roles is a complex UX.

**Comment by BaronRK** [comment]
谢谢 :)

**Comment by xenstar** [comment]
Thanks for bringing this. Additional to this. 

We need options like chatgpt, e.g promt starters, so when people open the agent page, they will know what type of questions they can ask. Especially if we have a lot of agents, we will forget what type of promt we should feed.

Currently only the default "just chat" has this feature.

![image](https://github.com/lobehub/lobe-chat/assets/55502651/e7ac18ac-8b8c-4e80-9463-e971878325ae)

**Comment by arvinxx** [comment]
@xenstar wait for https://github.com/lobehub/lobe-chat/discussions/891

**Comment by arvinxx** [comment]
@BaronRK You are right. We will improve this after we launch the document upload feature.
